### DISCUSSION TITLE
Download problem

### DISCUSSION THREAD

**Discussion by LilVega** [deniscerri/ytdlnis#558] [General]
Downloading:
Title: 
URL: https://youtu.be/zEY-qfi9r1o
Type: audio
Command: 
 -N "3" --trim-filenames "184" --no-mtime --sponsorblock-remove "music_offtopic" --parse-metadata "uploader:^(?P<uploader>.*?)(?:(?= - Topic)|$)" --parse-metadata "%(uploader,channel,creator,artist|null)s:%(uploader)s" --parse-metadata "%(artist,uploader|)s:^(?P<meta_album_artist>[^,]*)" --parse-metadata "%(album_artist,meta_album_artist|)s:%(album_artist)s" --parse-metadata "description:(?:Released on: )(?P<dscrptn_year>\d{4})" --parse-metadata "%(dscrptn_year,release_year,release_date>%Y,upload_date>%Y)s:(?P<meta_date>\d+)" --parse-metadata "%(album,title)s:%(meta_album)s" --extractor-args "youtube:player_client=default,mediaconnect,android" -f "ba/b" -x -P "/storage/emulated/0/Android/data/com.deniscerri.ytdl/files/downloads/2" -S "hasaud" --embed-metadata --embed-thumbnail --convert-thumbnails "jpg" --ppa "ThumbnailsConvertor:-qmin 1 -q:v 1 -vf crop=\"'if(gt(ih,iw),iw,ih)':'if(gt(iw,ih),ih,iw)'\"" -o "%(uploader).30B - %(title).170B.%(ext)s" "https://youtu.be/zEY-qfi9r1o" 

ERROR: [youtube] zEY-qfi9r1o: Sign in to confirm you’re not a bot. This helps protect our community. Learn more

How do i solve this

**Comment by teddysulaimanGL** [comment]
https://github.com/deniscerri/ytdlnis/issues/527

VPN. At the end of the day, it's unavoidable
